### PR TITLE
Update Cake to 0.28.0

### PR DESCRIPTION
**Pull request by gitfool**
Update Cake to ~~0.30.0~~0.28.0. Addresses some of the issues in #21.

### CODE REVIEW

File: build.ps1 [hunk 1]
```diff
@@ -5,11 +5,14 @@
 ##########################################################################
 
 <#
+
```
[the remark this answers is not in the document]
Comment (gitfool) [reply build.ps1]: Note that I simply dropped in the latest reference source from https://github.com/cake-build/resources/blob/develop/build.ps1. I'll revert that change.

**Comment by gitfool** [comment]
Re the branch, I think that happened because `master` is the default branch. Sorry about that.

File: Source/Cake.Codecov.Tests/Cake.Codecov.Tests.csproj [hunk 2]
```diff
@@ -3,11 +3,11 @@
     <TargetFrameworks>netcoreapp2.0;net46</TargetFrameworks>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Cake.Testing" Version="0.26.1" />
-    <PackageReference Include="FluentAssertions" Version="5.2.0" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.6.1" />
-    <PackageReference Include="xunit" Version="2.3.1" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.3.1" />
+    <PackageReference Include="Cake.Testing" Version="0.30.0" />
```
[the remark this answers is not in the document]
Comment (AdmiringWorm) [reply Source/Cake.Codecov.Tests/Cake.Codecov.Tests.csproj]: It have been mentioned a few times in the gitter channel, as well as it's mentioned in the guidelines: https://github.com/cake-contrib/Home/blob/master/Guidelines.md#supported-cake-version

File: build.ps1 (hunk 1 above)
[the remark this answers is not in the document]
Comment (AdmiringWorm) [reply build.ps1]: thank you

**Comment by AdmiringWorm** [comment]
> Re the branch, I think that happened because master is the default branch. Sorry about that.

Yeah, I thought as much. Should probably change it to develop, but meh. Too lazy 😄 

No worries though.

File: Source/Cake.Codecov.Tests/Cake.Codecov.Tests.csproj (hunk 2 above)
[the remark this answers is not in the document]
Comment (gep13) [reply Source/Cake.Codecov.Tests/Cake.Codecov.Tests.csproj]: For what it is worth, what @admiringworm is suggesting is what I would recommend as well.

**Pull request merged**